packages/components/src/Modal/Modal.rebuilt.tsx

```diff
@@ -128,12 +128,12 @@ export function ModalContent({ children }: ModalContainerProps) {
                 >
                   <div
                     ref={floatingRefs?.setFloating}
-                    role="dialog"
-                    tabIndex={0}
-                    aria-modal="true"
-                    aria-labelledby={modalLabelledBy}
-                    aria-label={accessibleName}
-                    {...getFloatingProps()}
+                    {...getFloatingProps({
+                      tabIndex: 0,
```

Comment:
do we even need to set the tabIndex ourselves?

floating has opinions on this and by using all their hooks, we've kinda put them in the driver's seat when it comes to focus management.